### DISCUSSION TITLE
开源之道（2016.6.4） - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#107] [Announcements]
# 开源之道（2016.6.4） - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2016/2016_6/2016_6_4/

**Comment by ZoomQuiet** [comment]
> 是也乎,(￣▽￣)

刚刚还在看 Nextcloud 没想到, 5年前就有相关事件了... 好象 PHPer 们就是比其它社区的人地图炮能力强?

不过, 类似什么是 Apache 的科普文章的确要长期进行,
不然, 原住民就会跳出来回血...
